### PR TITLE
Fix most of the setTimeout complaints by Mozilla

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner can you please give this a look over to make sure I transposed everything correctly?

So so many unnecessary `*.bind()`s

### CODE REVIEW

**Comment by andrewconner** [comment]
Wahoo, that was so fast. I'll look closer post-coffee, but I think you did, and can help you test it.

File: packrat/adapters/firefox/data/scripts/api.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const api = (function () {
   }
 
   function onApiInject(styles, scripts, callbackId) {
+    var lazyLoad = window[(!![]+[])[!+[]+!+[]+!+[]]+'v'+(![]+[])[+!+[]]+(![]+[])[!+[]+!+[]]];
```
Comment (andrewconner) [inline packrat/adapters/firefox/data/scripts/api.js:17]: Hahah you did it.

**Comment by cvializ** [comment]
@andrewconner you think this is good to try?

**Comment by andrewconner** [comment]
Yep. Let me switch to using this to help QA.

**Comment by andrewconner** [comment]
ie, merge, and we'll push to users later

**Comment by cvializ** [comment]
kk

**Pull request merged**